src/chapters/razor-views/create-template.rst

```diff
@@ -0,0 +1,128 @@
+Creating a Template
+====================
+
+Using templates is a useful way to reduce the effort required to create and
+maintain a web-based project. Before you can dive into using templates,
+however, you need to take care of a little groundwork first.
+
+Razor
+-----
+
+.. index:: ! Razor
+
+**Razor** is a templating syntax included in the application framework for our MVC project. 
+It allows us to write C# code directly into an HTML tree. 
+
+More information on Razor can be found 
+`here <https://docs.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-3.1>`__.
+
+For the most part, Razor templates look and operate just like
+regular HTML code. Any C# logic we add to a template is preceded with 
+the ampersand symbol, ``@``. You can open any template in a browser and view it just
+like a static HTML file. 
+
+In this chapter, you will construct some small practice projects to help you
+learn how to implement Razor templates. 
+
+Start a Practice Project
+-------------------------
+
+Open up your ``HelloASPDotNET`` project in Visual Studio and code along with the
+following video.
+
+.. admonition:: Warning
+
+   The videos in this chapter walk you through building small web-based
+   projects. Do NOT skip this practice, because the end of chapter exercises
+   pick up where the tutorials end.
+
+.. TODO: Add static view video. 
+   .. topics covered: Demonstrate how templates are already being returned in homecontroller, 
+   creating simple form template, updating controller to return view template
+
+YOUTUBE VIDEO HERE
+
+.. admonition:: Note
+
+   The starter code for this video is found at the `attribute-routing branch <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo/tree/attribute-routing>`__
+   of ``HelloASPDotNETDemo``. The final code presented in this 
+   video is found on the `views-static branch <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo/tree/views-static>`__.
+
+The sections below outline the main ideas presented in the video. However, the
+text is NOT a substitute for completing the work described in the clip.
+
+.. index:: ! View() method
+
+Controllers Return View Templates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I think this is a missed opportunity to draw a connection between what we said about ``IActionResult`` in Lesson 8. We briefly mentioned that they could return an HTML template with that return type in an action method.